build.gradle

```diff
@@ -130,6 +131,34 @@ sonarqube {
 }
 
 allprojects {
+
+    apply plugin: "com.diffplug.spotless"
+
+    spotless {
+        ratchetFrom 'origin/develop' // only format files which have changed since origin/main
+        format 'misc', {
+            // define the files to apply `misc` to
+            target '*.gradle', '*.md', '.gitignore'
+
+            // define the steps to apply to those files
+            trimTrailingWhitespace()
+            indentWithSpaces() // Takes an integer argument if you don't like 4
+            endWithNewline()
+        }
+        java {
+            // don't need to set target, it is inferred from java
+
+            // apply a specific flavor of google-java-format
+            googleJavaFormat('1.13.0').aosp().reflowLongStrings()
+
```

Comment:
AOSP style uses spaces for indentation. Although I'm completely in favor of this change, it will cause a lot of whitespace changes with the current style, which is TAB-based. And there might be objections from other team members regarding it. So, before introducing this change, we need to achieve a compromise across the team.

Another observations:
- AOSP uses double indentation (8 spaces) for wrapped function calls. This is fine for single wrapping without nested wraps. Long fluent call chains, with nested calls (lambdas) also wrapping, looks much less pretty. That's why we've using single indent (1 tab, equal to 4 spaces).
- We don't accept `if` without parentheses at all, AOSP allows one-line `if` statements
- We have much wider line hard wrap margin - 150 characters

Reply:
AOSP has been removed.